src/openloco/audio/audio.cpp

```diff
@@ -80,9 +80,12 @@ namespace openloco::audio
     static constexpr int32_t num_music_tracks = 29;
     static constexpr uint8_t no_song = 0xFF;
 
+    static loco_global<bool, 0x0050AED7> _backgroundMusicEnabled;
```

Comment:
What size is a `bool`?

Reply:
One byte. Am I missing something?

Reply:
`0x50AED7` is right in the middle of the config struct, though. We should probably access this from there instead.

Reply:
No, I just didn't know if it actually was one. I'm not sure if ever used booleans in OpenRCT2, so that's why I asked if there was a size issue with it.